zetaclient/orchestrator/bootstrap.go

```diff
@@ -71,7 +76,7 @@ func syncSignerMap(
 		presentChainIDs = make([]int64, 0)
 
 		onAfterAdd = func(chainID int64, _ interfaces.ChainSigner) {
-			logger.Std.Info().Msgf("Added signer for chain %d", chainID)
+			logger.Std.Info().Int64(logs.FieldChain, chainID).Msg("Added signer")
```

Comment:
not sure if it is worth hiding the `chain id` into attributes, but you can always check the behavior out in [Datadog link](https://app.datadoghq.com/logs?query=host%3Avalidator0-us-east-1-mainnet%20service%3Azetaclientd%20Added%20signer%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&context_event=AZJJfXCQAACrtQsVhJ5yRAAc&fromUser=true&messageDisplay=inline&refresh_mode=sliding&saved-view-id=2284468&storage=hot&stream_sort=time%2Cdesc&viz=&from_ts=1727460334418&to_ts=1728065134418&live=true)
The chain ids will be not showing.
![image](https://github.com/user-attachments/assets/30414bb1-8bad-4873-a377-50d17cba1a1d)

Reply:
- You can add any column to this table.
- It's better not to have any variables in the message because it's harder and more expensive to search / index logs
- I think your case works because here DD can do a prefix-based search.

cc @CharlieMc0